cms/templates/settings.html

```diff
@@ -140,8 +141,9 @@ <h2 class="title-2">${_("Credit Eligibility Requirements")}</h2>
 
                   % if 'proctored_exam' in credit_requirements:
                     <li class="field text is-not-editable" id="credit-proctoring-requirements">
-                      <label for="proctoring-requirements">${_("Successful Proctored Exam")}</label>
+                      <label>${_("Successful Proctored Exam")}</label>
                         % for requirement in credit_requirements['proctored_exam']:
+                          <label for="${requirement['name']}" class="sr">${_('Proctored Exam {counter}').format(counter=loop.index+1)}</label>
```

Comment:
This would be a little easier for translators to understand if you used `number` instead of `counter`.
```python
${_('Proctored Exam {number}').format(number=loop.index+1)}
```